.travis.yml

```diff
@@ -43,20 +43,22 @@ matrix:
     - php: hhvm
     - php: nightly
 
-before_script:
+before_install:
   - (phpenv config-rm xdebug.ini || exit 0)
   - mkdir -p ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d && echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   - composer selfupdate
   - composer config -q -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - composer global require phpunit/phpunit:@stable fabpot/php-cs-fixer --no-update
-  - composer global update --prefer-dist --no-interaction
+  - if [ "$TARGET" = "cs_dry_run" ]; then composer global require fabpot/php-cs-fixer --no-update; fi;
+  - if [ "$TARGET" = "test" ]; then composer global require phpunit/phpunit:@stable --no-update; fi;
   - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
-  - travis_wait composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
   - export PATH=$HOME/.local/bin:$PATH
-  - pip install -r Resources/doc/requirements.txt --user `whoami`
 
-script:
- - make $TARGET
+install:
+  - if [ "$TARGET" != "docs" ]; then composer global update --prefer-dist --no-interaction; fi;
```

Comment:
Lot of repetitions for no real benefits IMO.

Requiring two packages globally even if it's not used is not painful, could you please revert this part?

BTW, `.travis.yml` would be a generated file by dev-kit at the end: https://github.com/sonata-project/dev-kit/pull/31